### PR TITLE
Patch for FTP

### PR DESCRIPTION
**Pull request by walkerlala**

### CODE REVIEW

File: net-ftp/tnftp/tnftp-20151004.recipe [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+DESCRIPTION="NetBSD FTP client with several advanced features"
+HOMEPAGE="ftp://ftp.netbsd.org/pub/NetBSD/misc/tnftp/"
+SOURCE_URI="http://ftp.NetBSD.org/pub/NetBSD/misc/tnftp/tnftp-20151004.tar.gz"
+REVISION="1"
+CHECKSUM_SHA256="c94a8a49d3f4aec1965feea831d4d5bf6f90c65fd8381ee0863d11a5029a43a0"
+
+ARCHITECTURES="x86_64 x86 x86_gcc2"
+
+PROVIDES="
+	ntftp = $portVersion
```
Comment (korli) [inline net-ftp/tnftp/tnftp-20151004.recipe]: ntftp => tnftp

File: net-ftp/tnftp/tnftp-20151004.recipe [hunk 2]
```diff
@@ -0,0 +1,57 @@
...
+	runConfigure ./configure
+	make
+}
+
+INSTALL()
+{
+	make install
+}
+LICENSE="BSD (2-clause)"
+COPYRIGHT="2001-20015 The NetBSD Foundation, Inc. All rights reserved."
```
Comment (korli) [inline net-ftp/tnftp/tnftp-20151004.recipe]: Please move these two lines after HOMEPAGE.

Comment (korli) [inline net-ftp/tnftp/tnftp-20151004.recipe]: sorry *before* HOMEPAGE

File: net-ftp/tnftp/tnftp-20151004.recipe [hunk 3]
```diff
@@ -1,14 +1,16 @@
 SUMMARY="NetBSD luke FTP client"
 DESCRIPTION="NetBSD FTP client with several advanced features"
```
Comment (humdingerb) [inline net-ftp/tnftp/tnftp-20151004.recipe]: "You could elaborate the DESCRIPTION with what's at http://freecode.com/projects/tnftp/:

tnftp (formerly known as lukemftp) is a port of the NetBSD FTP client. It offers many enhancements over the traditional BSD FTP client, including
* command-line editing,
* command-line fetches of FTP and HTTP URLs (including via proxies),
* command-line uploads of FTP URLs,"

... etc

Mind the 80 char limit (compare to other recipes)

File: net-ftp/tnftp/tnftp-20151004.recipe [hunk 4]
```diff
@@ -1,14 +1,16 @@
...
 HOMEPAGE="ftp://ftp.netbsd.org/pub/NetBSD/misc/tnftp/"
+LICENSE="BSD (2-clause)"
+COPYRIGHT="2001-20015 The NetBSD Foundation, Inc. All rights reserved."
```
Comment (humdingerb) [inline net-ftp/tnftp/tnftp-20151004.recipe]: copyright before license, see "Ordering": https://github.com/haikuports/haikuports/wiki/HaikuPorter-Guidelines

File: net-ftp/tnftp/tnftp-20151004.recipe [hunk 5]
```diff
@@ -1,14 +1,16 @@
...
 SOURCE_URI="http://ftp.NetBSD.org/pub/NetBSD/misc/tnftp/tnftp-20151004.tar.gz"
 REVISION="1"
 CHECKSUM_SHA256="c94a8a49d3f4aec1965feea831d4d5bf6f90c65fd8381ee0863d11a5029a43a0"
```
Comment (humdingerb) [inline net-ftp/tnftp/tnftp-20151004.recipe]: Ordering again

File: net-ftp/tnftp/tnftp-20151004.recipe [hunk 6]
```diff
@@ -1,14 +1,16 @@
...
 
 ARCHITECTURES="x86_64 x86 x86_gcc2"
```
Comment (humdingerb) [inline net-ftp/tnftp/tnftp-20151004.recipe]: ordering: x86_gcc2  x86 x86_64

Comment (korli) [inline net-ftp/tnftp/tnftp-20151004.recipe]: @humdingerb sorry I didn't see your additional comments before merging.

**Pull request merged**